pymilvus/client/grpc_handler.py

```diff
@@ -123,14 +127,26 @@ def _setup_grpc_channel(self):
                              ('grpc.keepalive_time_ms', 55000)]
                 )
             else:
-                creds = grpc.ssl_channel_credentials(root_certificates=None, private_key=None, certificate_chain=None)
+                if self._client_pem_path != "" and self._client_key_path != "" and self._ca_pem_path != "" \
```

Comment:
Should these four parameters appear at the same time?

Reply:
Yes, I forget add server_name .